src/base_obs.c

```diff
@@ -174,10 +174,7 @@ static void update_obss(obss_t *new_obss)
   /* Copy over the time. */
   base_obss.tor = new_obss->tor;
 
-  /* Reset the `has_pos` flag. */
```

Comment:
I don't think the change is necessary as all accesses to `base_obss` should be mutex protected.